src/uproot_browser/tree_view.py

```diff
@@ -108,8 +106,6 @@ def render_tree_label(
     if is_cursor and has_focus:
         icon_label.stylize("reverse")
     if is_dir:
-        icon_label.stylize("green4") if expanded else icon_label.stylize(
-            "spring_green3"
-        )
+        icon_label.stylize("green4" if expanded else "spring_green3")
```

Comment:
@amangoel185 better to use the results, rather than relying on the short-circuiting behavior of a ternary if (otherwise, you could just use a normal if). Though I believe it does work.

Reply:
Okay!